src/plots/gl3d/scene.js

```diff
@@ -93,8 +111,22 @@ function render(scene) {
                 container: svgContainer
             });
         }
+
+        var eventData = makeEventData(xVal, yVal, zVal, trace, selection);
```

Comment:
Can we just inline this? It doesn't seem to get used anywhere else (as far as I see so far) and if it does, we can make it a function later.